rest/server_context.go

```diff
@@ -1155,13 +1155,13 @@ func (sc *ServerContext) updateCalculatedStats() {
 
 }
 
-func initClusterAgent(clusterAddress, clusterUser, clusterPass, certPath, keyPath, caCertPath string, timeoutSeconds *int) (*gocbcore.Agent, error) {
+func initClusterAgent(clusterAddress, clusterUser, clusterPass, certPath, keyPath, caCertPath string, CACertUnsetTlsSkipVerify bool, timeoutSeconds *int) (*gocbcore.Agent, error) {
```

Comment:
```suggestion
func initClusterAgent(clusterAddress, clusterUser, clusterPass, certPath, keyPath, caCertPath string, caCertUnsetTlsSkipVerify bool, timeoutSeconds *int) (*gocbcore.Agent, error) {
```

Reply:
Avoid capitalization in parameter names, although this name is subject to change anyway.